### PR TITLE
Dap 281 ledger UI

### PR DESCRIPTION
**Pull request by frank0170**

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -1 +1,2 @@
 @origyn-sa:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=YOUR_TOKEN
```
Comment (Motokoder) [inline .npmrc:2]: @3oltan is this going to break github actions? I mean worse than it's already broken?

File: package.json [hunk 2]
```diff
@@ -98,8 +98,8 @@
     "@mui/lab": "^5.0.0-alpha.109",
     "@mui/material": "^5.10.15",
     "@mui/x-date-pickers": "^5.0.9",
-    "@origyn-sa/mintjs": "^1.0.0-alpha.4",
-    "@origyn-sa/origyn-art-ui": "^1.2.12",
+    "@origyn-sa/mintjs": "^1.0.0-alpha.3",
```
Comment (Motokoder) [inline package.json:101]: Was this downgrade intentional?

**Review by Motokoder — CHANGES_REQUESTED** [review]

File: .npmrc (hunk 1 above)
Comment (frank0170) [reply .npmrc]: reversed

File: package.json (hunk 2 above)
Comment (frank0170) [reply package.json]: reversed, dont know why it happened

**Review by Motokoder — APPROVED** [review]

**Pull request merged**